Utilities/StrFmt.cpp

```diff
@@ -660,6 +660,17 @@ bool fmt::match(const std::string& source, const std::string& mask)
 	return true;
 }
 
+std::string fmt::aligned_padding(const std::string& input, usz length, bool right_align, char padding_char)
+{
+	if (input.length() < length)
```

Comment:
I think this would be easier to read as ```if (length > input.length())```
But that's probably just me

Reply:
No problem.
Fixed.